uploader.json

```diff
@@ -1167,12 +1290,16 @@
         "block": "/bin/ls /sys/block | awk '!/^ram|^\\.+$/ {print \"/dev/\" $1 \" unit s print\"}'",
         "ceph_osd_ec_profile_ls ": "/usr/bin/ceph osd erasure-code-profile ls",
         "ceph_socket_files": "/bin/ls /var/run/ceph/ceph-*.*.asok",
+        "docker_containers": "/usr/bin/docker ps -aq",
```

Comment:
This spec returns a massive amount of information on some systems.  Why are we doing this?